tests/modules/programs/yt-dlp/yt-dlp-simple-config.nix

```
@@ -0,0 +1,24 @@
+{ ... }:
+
+{
+  programs.yt-dlp = {
+    enable = true;
+    settings = {
+      embed-thumbnail = true;
+      embed-subs = true;
+      sub-langs = "all";
+      downloader = "aria2c";
+      downloader-args = "aria2c:'-c -x8 -s8 -k1M'";
+    };
+    extraConfig = ''
+      /home/user/.yt-dlp.conf
+    '';
+  };
+
```

No need to actually download the yt-dlp package in the test, so can stub it out:

```suggestion

    test.stubs.yt-dlp = { };

```